tpl/tplimpl/template_embedded.go

```diff
@@ -123,32 +123,47 @@ func (t *templateHandler) embedTemplates() {
 `)
 
 	t.addInternalTemplate("", "pagination.html", `{{ $pag := $.Paginator }}
-    {{ if gt $pag.TotalPages 1 }}
-    <ul class="pagination">
-        {{ with $pag.First }}
-        <li>
-            <a href="{{ .URL }}" aria-label="First"><span aria-hidden="true">&laquo;&laquo;</span></a>
-        </li>
-        {{ end }}
-        <li
-        {{ if not $pag.HasPrev }}class="disabled"{{ end }}>
-        <a href="{{ if $pag.HasPrev }}{{ $pag.Prev.URL }}{{ end }}" aria-label="Previous"><span aria-hidden="true">&laquo;</span></a>
-        </li>
-        {{ range $pag.Pagers }}
-        <li
-        {{ if eq . $pag }}class="active"{{ end }}><a href="{{ .URL }}">{{ .PageNumber }}</a></li>
-        {{ end }}
-        <li
-        {{ if not $pag.HasNext }}class="disabled"{{ end }}>
-        <a href="{{ if $pag.HasNext }}{{ $pag.Next.URL }}{{ end }}" aria-label="Next"><span aria-hidden="true">&raquo;</span></a>
-        </li>
-        {{ with $pag.Last }}
-        <li>
-            <a href="{{ .URL }}" aria-label="Last"><span aria-hidden="true">&raquo;&raquo;</span></a>
-        </li>
-        {{ end }}
-    </ul>
-    {{ end }}`)
+{{ if gt $pag.TotalPages 1 }}
```

Comment:
My assumption is that this would be a breaking change, but really this needs to be wrapped in `<nav role="navigation" aria-label="Pagination Navigation">`, but I appreciate this is a breaking change. Only mentioning this here as a comment that I will likely add to new docs...